apps/desktop/src/lib/commit/CommitList.svelte

```diff
@@ -49,20 +49,30 @@
 	const reorderDropzoneManagerFactory = getContext(ReorderDropzoneManagerFactory);
 	const gitHost = getGitHost();
 
+	enum LineSpacer {
+		Remote = 'remote-spacer',
+		Local = 'local-spacer',
+		LocalAndRemote = 'local-and-remote-spacer'
+	}
```

Comment:
Where should this enum belong?

Reply:
I'd say it is, just to mitigate the risk of typo-bugs since the `CommitData` id's are free-form strings. The compiler wouldn't catch typos otherwise.

Alternatively I could just turn this into simple string constants, but I initially chose enum for some added context.

> where should this enum belong?
>

Good question, I didn't immediately find a right place for it other than this module as it's only specific to the `CommitList` component. What do you think?